calypso/csadmin/clicontracts/read_test.sh

```diff
@@ -0,0 +1,58 @@
+# This method should be called from the calypso/csadmin/test.sh script
+
+testContractRead() {
+    run testContractReadInvoke
```

Comment:
why do we need an extra wrapper? can we not just do run testContractLTSInvoke inside test.sh?